### PR TITLE
Changing font files

### PR DESCRIPTION
**Pull request by mmlumba**
This will add the bolder fonts to the applications as per issue #9

### CODE REVIEW

File: app/assets/stylesheets/_fonts.scss [hunk 1]
```diff
@@ -9,6 +9,14 @@
 }
 
 @font-face {
+  font-family: 'Source Sans Pro';
+  font-weight: 700;
```
Comment (cmgiven) [inline app/assets/stylesheets/_fonts.scss]: Can we get a `font-style` here for completeness?

**Comment by mmlumba** [comment]
@cmgiven do you know where the licenses for the fonts are? I addressed your other change regarding the `font-style`.

**Comment by cmgiven** [comment]
Where we got the files from, most likely. Source Sans is SIL and on GutHub here: https://github.com/adobe-fonts/source-sans-pro

I suspect Shane has thoughts on style changes, but my general sense is we override the styles in the edge apps with comments indicating a candidate for Commons. Then we do a periodic review to consolidate those styles here.

**Comment by cmgiven** [comment]
*GitHub

**Comment by mmlumba** [comment]
@cmgiven What I meant in my previous comment was this: How often do we update the gem version number of the caseflow gem with every PR that gets merged into caseflow-commons? I wasn't sure if the gem needed to be updated from 0.15 to 0.16 for a small font change.

**Comment by cmgiven** [comment]
Ah, gotcha. I think we tick the version every time. It doesn't matter at this point, as we're requiring it via ref, but once we get our own gem server up and running, as I understand it, we'd use those version numbers to control when we pull changes into each app.

**Comment by KHarshawat** [comment]
@mmlumba can you provide screenshots of what the updated fonts look like in the app? Perhaps a few pages in certification and in eFolder express?

**Comment by mmlumba** [comment]
I added a sample screenshot in issue #9 (from eFolder) but I'll add more from the app to this PR.

**Comment by mmlumba** [comment]
@KHarshawat Just as an FYI too if you want these design changes to be applied to all the apps immediately then we would need to change the `Gemfile` on those apps to reference the correct commit number.

**Comment by mmlumba** [comment]
@KHarshawat Here's a bunch of screenshots from the different apps.
eFolder help page:
screen shot 2017-01-03 at 1 48 55 pm

Certification: what's new?
screen shot 2017-01-03 at 9 16 24 pm

Certification: appeal ready to be certified
screen shot 2017-01-03 at 9 20 31 pm

Certification: appeal with VBMS error
screen shot 2017-01-03 at 9 21 54 pm

Certification: appeal not ready
screen shot 2017-01-03 at 9 22 02 pm

**Comment by KHarshawat** [comment]
Approved!

**Comment by mmlumba** [comment]
@astewarttistatech @kierachell Instructions for testing:
Find the line `gem "caseflow"`within the `Gemfile` for any app (certification, efolder, feedback) and modify the line to look like the following:
`gem "caseflow", git: "https://github.com/department-of-veterans-affairs/caseflow-commons", branch: master`
Run `bundle install`
View the app in Dev and verify the app looks similar to above screenshots in PR
Revert the `gem "caseflow"` line back to its original state

**Pull request merged**